src/app/views/canvas/mark_editor.tsx

```diff
@@ -1558,6 +1583,18 @@ export class SingleMarkView
         //     };
         //   }
         //   break;
+        case "rectangle-zoom":
+          {
+            mode = "rectangle";
+            onCreate = (x1, y1, x2, y2) => {
```

Comment:
I'm guessing this is the case, but I am curious if this feature works when the chart-editor or glyph-editor views are already zoomed or panned.

Are the `x1`, `y1`, ... parameters normalized to a global zoom factor and translation? I was trying to find in the code where these parameters are computed. Anyways, I figure this is already taken care of, but wanted to check.